docs/team/wonyeji.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: Won Ye Ji's Project Portfolio Page
+---
+
+### Project: Thunder Cats
+
+Thunder Cats is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
```

Comment:
We may need to standardise the product overview, but this can be discussed later. This should be fine for now.